findimports.py

```diff
@@ -929,84 +935,98 @@ def quote(s):
-                      dest='action', const='printUnusedImports',
-                      help='print unused imports')
-    parser.add_option('-a', '--all', action='store_true',
-                      dest='all_unused',
-                      help="don't ignore unused imports when there's a comment"
-                           " on the same line (only affects -u)")
-    parser.add_option('--duplicate', action='store_true',
-                      dest='warn_about_duplicates',
-                      help='warn about duplicate imports')
-    parser.add_option('--ignore-stdlib', action='store_true',
-                      dest='ignore_stdlib',
-                      help="ignore the imports of modules from the Python"
-                           " standard library")
-    parser.add_option('-v', '--verbose', action='store_true',
-                      help='print more information (currently only affects'
-                           ' --duplicate)')
-    parser.add_option('-N', '--noext', action='store_true',
-                      help='omit external dependencies')
-    parser.add_option('-p', '--packages', action='store_true',
-                      dest='condense_to_packages',
-                      help='convert the module graph to a package graph')
-    parser.add_option('-l', '--level', type='int',
-                      dest='packagelevel',
-                      help='collapse subpackages to the topmost Nth levels')
-    parser.add_option('-c', '--collapse', action='store_true',
-                      dest='collapse_cycles',
-                      help='collapse dependency cycles')
-    parser.add_option('-T', '--tests', action='store_true',
-                      dest='collapse_tests',
-                      help="collapse packages named 'tests' and 'ftests'"
-                           " with parent packages")
-    parser.add_option('-w', '--write-cache', metavar='FILE',
-                      help="write a pickle cache of parsed imports; provide"
-                           " the cache filename as the only non-option"
-                           " argument to load it back")
-    parser.add_option('-I', '--ignore', metavar='FILE', action="append",
-                      help="ignore a file or directory;"
-                           " this option can be used multiple times."
-                           " Default: ['venv']")
+    parser = argparse.ArgumentParser(
+        usage='%(prog)s [action] [options] [filename|dirname ...]',
+        prog=progname, description=description)
+    
+    parser.add_argument(
+        'filenames', metavar='filename|dirname', nargs='*', default=['.'],
+        help='The files and/or directories to inspect, default: "."')
+    
+    actions = parser.add_argument_group('actions',
+                                        description='Exactly one of these'
+                                                    ' actions will be performed'
+                                                    ' (default: --imports)')
```

Comment:
> findimports.py:948:80: E501 line too long (80 > 79 characters)

I don't mind if the fix for this is a `max-line-length = 80` addition to the `[flake8]` section in `setup.cfg` instead of squeezing the text column even more narrowly.

Personally I've started to prefer the Black style of 
```suggestion
    actions = parser.add_argument_group(
        'actions',
        description='Exactly one of these actions will be performed (default: --imports)'
    )
```

(which probably doesn't fit in 80 chars either, I can't see the width in the GitHub comment box.)

Reply:
I've gone with:
```
    actions = parser.add_argument_group(
        'actions',
        description='Exactly one of these actions will be performed'
                    ' (default: --imports)')
```
Which IMHO is a good compromise on the vertical space.